src/views/shared/client/utils.tsx

```diff
@@ -86,18 +87,36 @@ export function ReplaceNewlines(text: string): string {
     .join("");
 }
 
-export function LoadCardImageIntoElement(
```

Comment:
If the code base weren't garbage this wouldn't had a function comment for you to update, but it didn't :(

Can you add a function comment explaining that the returned promise resolves after the card image is loaded?

Reply:
What do you mean by "default image", and "if necessary"? I think something like this would be clearer:
// Resolves after the card image is loaded into the element.

Reply:
Comment added

Reply:
This was lost, like the other fix